### ISSUE TITLE
Create fundraising FAQ for PR team, social media teams, SUMO; etc

### ISSUE THREAD

**Issue by valianttry** [mozilla/donate.mozilla.org#871]
During the course of our EOY fundraising campaign, people across the org sometimes have to field questions about our campaign, and how fundraising fits into our organizational structure / plan. 

We already have probably 75% of an FAQ done, here: https://wiki.mozilla.org/Donate

We may need to add a few questions like:
*How long has Mozilla been fundraising?
*What % of Mozilla's donation revenue goes to overhead?
etc.

However, we may need to add some content about the upcoming release of Mozilla's financials and have FAQs to answer how fundraising fits into that. (e.g. a small # of bloggers or writers sometimes say things like, "Revenue is down, is Firefox getting desperate and begging now with this new fundraising campaign?") We should anticipate what Qs might arise from the financials publicity, if any. (Last year was really pretty quiet on that front.) EJ has historic knowledge of this as well as Justin O.

**Comment by valianttry** [comment]
cc @edrushka

**Comment by KevZawacki** [comment]
cc @HPaulJohnson 

Thanks for the background, @valianttry. Below are suggested additions to the FAQ. I've tagged you in parts that need fact-checking / more information. Also: do you envision we simply add these to the wiki and then circulate among relevant folks?

FYI, I also have an email in to Justin to see if he has suggestions/additions.

**Don't Mozilla products, like Firefox, earn income?** 

Firefox is maintained by the Mozilla Corporation, a wholly-owned subsidiary of the Mozilla Foundation. While Firefox does produce revenue — chiefly through search partnerships — this earned income is reinvested back into the Corporation. 

The Mozilla Foundation's education and public awareness efforts, which span several continents and reach millions of people, are made possible through philanthropic donations.   

**Who donates to Mozilla?**

Last year, more than 200,000 individuals gave to Mozilla to protect and improve the Web. The Mozilla Foundation also receives support in the form of grants from other major foundations, like Knight, MacArthur and Ford. @valianttry [do we want to include specific numbers?]

**How long has Mozilla been fundraising?**

Mozilla first began fundraising in 2011, to support its new Foundation initiatives. @valianttry 

**What % of Mozilla's donation revenue goes to overhead?**

@valianttry

**Comment by KevZawacki** [comment]
Update: Justin has approved.

**Comment by valianttry** [comment]
There were over 380,000 donors in 2014 (not 200K) I wouldn't include specific breakdown of our financials. I would just say "for more information, you can view our financials here" and link to our 990 or whatever we have on the web. We can also create a short link to this detailed blog post about our revenue generation activities Denelle and Mark worked on last year: https://blog.mozilla.org/blog/2014/12/19/building-a-healthy-web-to-hand-to-future-generations/

I would not put the "what % of Mozilla's revenue goes to overhead?" in any public place. We should have an internal response ready if needed though.

Basically "overhead" is a really bogus measure of effectiveness, so we don't want to play into this fallacy. See: https://www.guidestar.org/Articles.aspx?path=/rxa/news/news-releases/2014/2014-10-21-charities-urged-to-crush-overhead-myth.aspx

**Comment by KevZawacki** [comment]
@valianttry Thanks! I've just updated our new questions and ported them into the FAQ wiki. I'll share these updates with Michaela and team, too. 

Closing.
